src/ot_orchestration/utils/dataproc_platform.py

```diff
@@ -0,0 +1,225 @@
+"""Utility functions for working with Dataproc clusters in the Platform project."""
+
+from collections.abc import Sequence
+
+from airflow.providers.google.cloud.operators.dataproc import (
+    DataprocCreateClusterOperator as _DataprocCreateClusterOperator,
+)
+from airflow.providers.google.cloud.operators.dataproc import (
+    DataprocSubmitJobOperator as _DataprocSubmitJobOperator,
+)
+from google.cloud.dataproc_v1 import Cluster, JobReference
+from google.cloud.dataproc_v1.types.jobs import Job, JobPlacement, SparkJob
+
+from ot_orchestration.utils import random_id
+from ot_orchestration.utils.common import GCP_PROJECT_PLATFORM, GCP_REGION
+from ot_orchestration.utils.dataproc import ClusterGenerator
+from ot_orchestration.utils.labels import Labels
+
+
+class DataprocCreateClusterOperator(_DataprocCreateClusterOperator):
```

Comment:
Convince me that it is a good idea to have the name overwritten.

Reply:
It's not, I was doubtful about it :D

Reply:
Let's keep it for now, there should not be any issues.

Reply:
I changed it in the suggestions commit. :+1: